File: {{cookiecutter.project_slug}}/src/main/java/{{cookiecutter.group_id.split('.')[0]}}/{{cookiecutter.group_id.split('.')[1]}}/{{cookiecutter.mod_main_package_name}}/network/packets/ServerPacketName.java
package {{cookiecutter.group_id}}.{{cookiecutter.modid}}.network.packets;

import io.netty.buffer.ByteBuf;

public class ServerPacketName extends AbstractPacketServer {

    @Override
    public void write(ByteBuf data) throws IndexOutOfBoundsException {
        data.writeBoolean(true);
    }
}

File: src/main/java/org/dreamfinity/template/network/packets/ServerPacketName.kt
package org.dreamfinity.template.network.packets

import io.netty.buffer.ByteBuf

class ServerPacketName : AbstractPacketServer() {
    @Throws(IndexOutOfBoundsException::class)
    override fun write(data: ByteBuf?) {
        data!!.writeBoolean(true)
    }
}